.gitignore

```diff
@@ -1,2 +1,16 @@
+images/
 node_modules/
```

Comment:
You will also need to delete node_modules and commit in order to get rid of it from source code.